foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -150,7 +151,7 @@ private static void duplicateServiceCombConfigToCse(ConcurrentCompositeConfigura
   }
 
   public static DynamicWatchedConfiguration createConfigFromConfigCenter(Configuration localConfiguration) {
-    if (localConfiguration.getProperty(configCenterUrlKey) == null) {
+    if ((localConfiguration.getProperty(configCenterUrlKey) == null) && (localConfiguration.getProperties((configApolloUrlKey)) == null )) {
```

Comment:
each time, we add a new IMPL.
we need to modify this place.
it's very bad.

Reply:
OK, lets remove this check here and do the check after we get the configCenterSource.